cloud-sql/postgres/sqlalchemy/requirements.txt

```diff
@@ -1,4 +1,4 @@
 Flask==1.1.2
 SQLAlchemy==1.3.20
-pg8000==1.16.6
+pg8000==1.16.5
```

Comment:
Can you add a comment here that we shouldn't update this past 1.16.6 until this issue is resolved? https://github.com/tlocke/pg8000/commit/3a2e7439ae3613367ec231218d7e0f541466d1e5#commitcomment-43174891

Reply:
done